source/frontend/src/features/leases/list/LeaseFilter/LeaseFilter.tsx

```diff
@@ -208,8 +209,8 @@ export const LeaseFilter: React.FunctionComponent<React.PropsWithChildren<ILease
                 </Col>
               </Row>
             </Col>
-            <Col xs="5">
-              <SectionField label="Expiry date" labelWidth="2">
+            <Col xl="5" xs="12">
+              <SectionField className="pb-0" label="Expiry date" labelWidth={{ xl: '2' }}>
```

Comment:
This keeps the filter fields vertically-aligned - may require some tech debt to resolve.